source/sync/go-to-production/compact-disk.txt

```diff
@@ -4,62 +4,77 @@
+   on your Atlas App Services app. To do so, set the :guilabel:`Client Max Offline Time`
+   in the Sync Settings through the App Services UI. For more information, refer to
+   :ref:`set-the-client-maximum-offline-time`.
 
    .. note:: Wait for Compaction
-   
-      If you are enabling compaction for the first time, you will need to wait for 
-      the compaction to happen before continuing. You can file a ticket with 
+
+      If you are enabling compaction for the first time, you will need to wait for
+      the compaction to happen before continuing. You can file a ticket with
       support if you want to ensure compaction has completed.
 
 #. Perform a Rolling Resync
 
-   Perform a rolling resync of your Atlas cluster, a process explained in 
+   Perform a rolling resync of your Atlas cluster, as described in
    :manual:`Resync a Member of a Replica Set <tutorial/resync-replica-set-member/>`.
-   
-   The commands to do this are not available with Atlas. Instead, you can 
+
+   The commands to do this are not available through Atlas. Instead, you can
    file a ticket with support to have this process done for you.
 
-An Alternate Approach
----------------------
+Use ``Compact()``
+-----------------
+
+Another step that you can take to reclaim Atlas volume space is to compact the
+indexes on the Atlas cluster.
+
+1. Test the primary failover process (recommended). Because this procedure
+   involves performing a failover on the primary node, we highly recommend that
+   you test whether your cluster can handle a primary failover before
+   proceeding.
+
+   For more information on performing this test and its prerequisites, refer to
+   :atlas:`Test Primary Failover </tutorial/test-resilience/test-primary-failover/>`.
+
+#. Run the :manual:`compact() </reference/command/compact/>` command on one of the
+   secondary nodes, and then on the other secondary nodes.
+
+#. Perform a failover on the primary mode, then run ``compact()`` on the primary
+   node.
 
-A final approach you can take for reclaiming some of your Atlas volume space
-is to run the ``compact()`` command. You run it on one of the secondary nodes
-and then the other secondary node(s). Finally, you perform a failover on the
-primary mode and then run ``compact()``. For more information on this approach,
-support-enabled customers can refer to `How to use the compact() command in
-Atlas <https://support.mongodb.com/article/000019602>`__. Contact
-:atlas:`Support </support>` for access.
+For more information on this approach, support-enabled customers can refer to
+`How to use the compact() command in Atlas
+<https://support.mongodb.com/article/000019602>`__.
+Contact :atlas:`Support </support>` for access.
 
-.. note:: 
+.. note::
 
-   The ``compact()`` command may not reduce the volume size as much as 
-   performing a rolling resync. 
+   The ``compact()`` command may not reduce the volume size as much as
+   performing a rolling resync.
```

Comment:
If you do take my suggestion above about making the "Perform a rolling resync" step "File a support ticket" (since users can't actually do this themselves), then I think we should reword this:
```suggestion
   filing a support ticket to have a rolling resync performed.
```